test/model-tests/data-service.json

```diff
@@ -10633,8 +10633,32 @@
         "ALN",
         "ALNR",
         "ALN",
+        "ANALDEP",
+        "DEFECATIONMUSCLES",
```

Comment:
I'm not clear why this file changed so much.

Reply:
> I'm not clear why this file changed so much.

Ah, I think this has to do with the unc-31 dataset being "complete animal" now?

Reply:
yes that's it

Reply:
btw to rebuild db can "npm run populate-database" and "npm run build-prod" which I think includes the former command as part of the things it does, but to be sure you'd want to do both.

Reply:
Thx. I don't think you need to do `npm run build-prod` if you're only hosting it via `npm run watch`, but I'm not 100% sure.

Reply:
I was rebuilding the db (only when necessary) by doing:

```
sh scripts/setup_config.sh "$DBNAME" "$DBUSER" "$PASSWD" | sudo mysql
```

but that should only be necessary if the database schema changed.